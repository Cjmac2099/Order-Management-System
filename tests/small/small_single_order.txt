KE7BYE2XB1
71.5
END
