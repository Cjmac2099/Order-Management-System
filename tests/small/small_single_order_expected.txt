Loading catalog...
Enter 'END' to finish order.
SKU> 
SKU matches product Canned Apricots packed in syrup or water, price per pound: $2.24
Quantity>
SKU> 
Printing order summary:
71.50lbs Canned Apricots packed in syrup or water: $159.89
